.circleci/config.yml

```
@@ -1,494 +1,55 @@
-          paths:
-            - ~/.gradle
-          key: gradle-v2-{{ .Branch }}-{{ checksum "build.gradle" }}
+      # run tests!
+      - run: gradle test
 
+# Orchestrate jobs using workflows
+# See: https://circleci.com/docs/guides/orchestrate/workflows/ & https://circleci.com/docs/reference/configuration-reference/#workflows
 workflows:
-  version: 2
-  build:
+  sample: # This is the name of the workflow, feel free to change it to better match your workflow.
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)